rebus/chain.json

```diff
@@ -0,0 +1,225 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "rebus",
+    "website": "https://www.rebuschain.com/",
+    "status": "live",
+    "network_type": "mainnet",
+    "pretty_name": "Rebus",
+    "chain_id": "reb_1111-1",
+    "bech32_prefix": "rebus",
```

Comment:
please register this prefix to https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
That should be good to go https://github.com/satoshilabs/slips/pull/1407